### ISSUE TITLE
How do I see test loss?

### ISSUE THREAD

**Issue by aletote** [karpathy/minGPT#56]
That.

**Comment by gokceneraslan** [comment]
<img width="670" alt="image" src="https://user-images.githubusercontent.com/1140359/109685383-35317a80-7b4f-11eb-849d-473ba39e6cb0.png">

It is printed after each epoch if you set the testset parameter of the `Trainer()`
